src/locales/nl.json

```
@@ -0,0 +1,172 @@
+    "identities.messages.1050011": "Bevestig back-up herstelcodes",
+    "identities.messages.1050012": "Beveiligingssleutel toevoegen",
+    "identities.messages.1050013": "Naam van de beveiligingssleutel",
+    "identities.messages.1050014": "<del>Gebruikt</del>",
+    "identities.messages.1050015": "{secrets_list}",
+    "identities.messages.1050016": "Deze methode uitschakelen",
+    "identities.messages.1050017": "Dit is uw geheime Authenticator-app. Gebruik het als u de QR-code niet kunt scannen.",
+    "identities.messages.1050018": "Beveiligingssleutel verwijderen \"{display_name}\"",
+    "identities.messages.1060001": "U heeft met succes uw account hersteld. Wijzig alstublieft uw wachtwoord of stel een alternatieve aanmeldingsmethode in (bijv. sociale aanmelding) binnen de komende {privileged_session_expires_at_unix_until_minutes} minuten.",
+    "identities.messages.1060002": "Er is een e-mail met een herstelkoppeling verzonden naar het opgegeven e-mailadres. Als u geen e-mail heeft ontvangen, controleer dan de spelling van het e-mailadres en zorg ervoor dat u het e-mailadres gebruikt waarmee u zich heeft geregistreerd.",
+    "identities.messages.1060003": "Er is een e-mail met een herstelcode verzonden naar het opgegeven e-mailadres. Als u geen e-mail heeft ontvangen, controleer dan de spelling van het e-mailadres en zorg ervoor dat u het e-mailadres gebruikt waarmee u zich heeft geregistreerd.",
+    "identities.messages.1070001": "Wachtwoord",
+    "identities.messages.1070002": "{title}",
+    "identities.messages.1070003": "Opslaan",
+    "identities.messages.1070004": "ID",
+    "identities.messages.1070005": "Verzenden",
+    "identities.messages.1070006": "Verificatiecode controleren",
+    "identities.messages.1070007": "E-mail",
+    "identities.messages.1070008": "Code opnieuw verzenden",
+    "identities.messages.1070009": "Doorgaan",
+    "identities.messages.1070010": "Herstelcode",
+    "identities.messages.1070011": "Verificatiecode",
+    "identities.messages.1070012": "Registratiecode",
+    "identities.messages.1070013": "Aanmeldingscode",
+    "identities.messages.1080001": "Er is een e-mail met een verificatielink naar het opgegeven e-mailadres gestuurd. Als je geen e-mail hebt ontvangen, controleer dan de spelling van het e-mailadres en zorg ervoor dat je het e-mailadres hebt gebruikt waarmee je hebt geregistreerd.",
+    "identities.messages.1080002": "Je hebt succesvol je e-mailadres geverifieerd.",
+    "identities.messages.1080003": "Er is een e-mail met een verificatiecode naar het opgegeven e-mailadres gestuurd. Als je geen e-mail hebt ontvangen, controleer dan de spelling van het e-mailadres en zorg ervoor dat je het e-mailadres hebt gebruikt waarmee je hebt geregistreerd.",
+    "identities.messages.4000001": "{reden}",
+    "identities.messages.4000002": "Eigenschap {eigenschap} ontbreekt.",
+    "identities.messages.4000003": "lengte moet >= {min_lengte} zijn, maar is {werkelijke_lengte}",
```

```suggestion
    "identities.messages.4000003": "lengte moet >= {min_length} zijn, maar is {actual_length}",
```
https://github.com/ory/elements/blob/main/src/locales/de.json#L76